Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -123,6 +123,9 @@ lemma fin3_def (P : Fin 3 → R) : ![P x, P y, P z] = P := by
 lemma fin3_def_ext (X Y Z : R) : ![X, Y, Z] x = X ∧ ![X, Y, Z] y = Y ∧ ![X, Y, Z] z = Z :=
   ⟨rfl, rfl, rfl⟩
 
+lemma comp_fin3 {S} (f : R → S) (X Y Z : R) : f ∘ ![X, Y, Z] = ![f X, f Y, f Z] := by
+  ext i; fin_cases i <;> rfl
```

Comment:
This is `FinVec.map_eq`

Reply:
Thanks! It seems it can't be directly used to rewrite to the desired form, or to replace `comp_fin3` in a `simp` call, so I just changed the proof of `comp_fin3` to use it.